src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,100 @@
+            I = 0,
+            D = 0;
+    private static final CANSparkMax ANGLE_MOTOR_MASTER = new CANSparkMax(ANGLE_MOTOR_MASTER_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private static final CANSparkMax ANGLE_MOTOR_FOLLOWER = new CANSparkMax(ANGLE_MOTOR_FOLLOWER_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private static final CANSparkMax ELEVATOR_MOTOR_MASTER = new CANSparkMax(ELEVATOR_MOTOR_MASTER_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private static final CANSparkMax ELEVATOR_MOTOR_FOLLOWER = new CANSparkMax(ELEVATOR_MOTOR_FOLLOWER_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private static final PIDController PID_CONTROLLER = new PIDController(P, I, D);
+    private static final int
+            ANGLE_ENCODER_DIRECTION = 0,
+            ELEVATOR_ENCODER_DIRECTION = 0;
+    private static final CANcoder ANGLE_ENCODER = new CANcoder(ANGLE_ENCODER_ID);
+    private static final TalonFX ELEVATOR_ENCODER = new TalonFX(ELEVATOR_ENCODER_ID);
+
+    public enum ArmState {
+        DEFAULT(Rotation2d.fromDegrees(0), 0),
+        TAKE_HIGH_CONE(Rotation2d.fromDegrees(60), 10),
+        TAKE_GROUND_CONE(Rotation2d.fromDegrees(0), 0),
+        PLACE_LOW_CONE(Rotation2d.fromDegrees(20), 3),
+        PLACE_MEDIUM_CONE(Rotation2d.fromDegrees(50), 7),
+        PLACE_HIGH_CONE(Rotation2d.fromDegrees(70), 10);
+
+        final Rotation2d angle;
+        final double elevatorPosition;
+
+        ArmState(Rotation2d angle, double elevatorPosition) {
+            this.angle = angle;
+            this.elevatorPosition = elevatorPosition;
+        }
+    }
+
+
+    static {
+        configureAngleMotors();
+        configureElevatorMotors();
+    }
+
+    private static void configureAngleMotors() {
+        ANGLE_MOTOR_MASTER.restoreFactoryDefaults();
+        ANGLE_MOTOR_MASTER.setInverted(ANGLE_MOTOR_MASTER_INVERTED);
+        ANGLE_MOTOR_MASTER.setIdleMode(ANGLE_MOTORS_IDLE_MODE);
+        ANGLE_MOTOR_MASTER.enableVoltageCompensation(VOLTAGE_COMPENSATION_SATURATION);
+
+        ANGLE_MOTOR_FOLLOWER.restoreFactoryDefaults();
+        ANGLE_MOTOR_FOLLOWER.setInverted(ANGLE_MOTOR_FOLLOWER_INVERTED);
+        ANGLE_MOTOR_FOLLOWER.setIdleMode(ANGLE_MOTORS_IDLE_MODE);
+        ANGLE_MOTOR_FOLLOWER.enableVoltageCompensation(VOLTAGE_COMPENSATION_SATURATION);
+    }
+
+    private static void configureElevatorMotors() {
+        ELEVATOR_MOTOR_MASTER.restoreFactoryDefaults();
+        ELEVATOR_MOTOR_MASTER.setInverted(ELEVATOR_MOTOR_MASTER_INVERTED);
+        ELEVATOR_MOTOR_MASTER.setIdleMode(ELEVATOR_MOTORS_IDLE_MODE);
+        ELEVATOR_MOTOR_MASTER.enableVoltageCompensation(VOLTAGE_COMPENSATION_SATURATION);
+
+        ELEVATOR_MOTOR_FOLLOWER.restoreFactoryDefaults();
+        ELEVATOR_MOTOR_FOLLOWER.setInverted(ELEVATOR_MOTOR_FOLLOWER_INVERTED);
+        ELEVATOR_MOTOR_FOLLOWER.setIdleMode(ELEVATOR_MOTORS_IDLE_MODE);
+        ELEVATOR_MOTOR_FOLLOWER.enableVoltageCompensation(VOLTAGE_COMPENSATION_SATURATION);
+    }
```

Comment:
I advise putting similar configurations next to each other (ie. MASTER_MOTOR.setInverted next to FOLLOWER_MOTOR.setInverted)